dev-python/pytest-faulthandler/pytest-faulthandler-2.0.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Pytest plugin that activates the fault handler module for tests"
+HOMEPAGE="https://github.com/pytest-dev/pytest-faulthandler"
+SRC_URI="https://github.com/pytest-dev/pytest-faulthandler/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	>=dev-python/pytest-5.0[${PYTHON_USEDEP}]
+"
```

Comment:
There are no tests here: https://github.com/pytest-dev/pytest-faulthandler

It depends on pytest not for testing itself but because it is a pytest plugin, so I suppose this should be a RDEPEND.